src/main/java/seedu/address/logic/commands/GMapsCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import java.util.List;
+import java.util.Objects;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.Address;
+import seedu.address.model.person.ReadOnlyPerson;
+
+/**
+ * Selects a person identified using it's last displayed index from the address book.
+ */
+public class GMapsCommand extends Command {
+
+    public static final String COMMAND_WORD = "gmaps";
+    public static final String COMMAND_ALIAS = "g";
+    public static final String COMMAND_SECONDARY_ONE = "map";
+    public static final String COMMAND_SECONDARY_TWO = "maps";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Opens a Google Maps view of a person’s address.\n"
+            + " If you specify an address, you can find directions from the address to that person's address.\n"
+            + "Format: " + COMMAND_WORD + " INDEX [a/ADDRESS]\n"
+            + "Example: " + COMMAND_WORD + " 1 \n"
+            + "         " + COMMAND_WORD + " 1 a/Blk 123, Yishun 75";
```

Comment:
can the gmap command support zipcode also?
haha, I think it will be excellent as we can also easily search zipcode in google map!